.github/workflows/test_codebase.yml

```diff
@@ -1,8 +1,9 @@
-name: "Test"
+name: "Testing Codebase"
 
-on:
+on: 
+  workflow_dispatch:
   push:
-    branches: [main,github-actions-test]
+    branches: [main,github-actions-test,ghaction-speedup-iss215]
```

Comment:
This should not come into the main branch, please remove from this pull request

Reply:
I have made the change. Sorry for this being left over!